app/furniture/marketplace/cart.rb

```diff
@@ -12,6 +12,7 @@ class Cart < ApplicationRecord
     has_many :cart_products, dependent: :destroy, inverse_of: :cart
     has_many :products, through: :cart_products, inverse_of: :carts
     has_one :checkout, inverse_of: :cart
+    has_one :order, inverse_of: :cart, class_name: "Order"
```

Comment:
I don't think the `class_name` is actually useful here 🤦

Reply:
what happens if you remove it? what breaks?

Reply:
I don't think anything will, I just added it sillilly.

Reply:
Remove it! 🪄 ✨ 🔥 it! 😄

Reply:
IT DEAD